Tech-Lab-On-Campus/README.md

```diff
@@ -22,40 +22,29 @@ The idea of this lab is to offer exposure to the RabbitMQ messaging framework, p
 ## Setting Up Our Environment
 For this project, we're going to leverage the use of Docker to create a helpful development environment for all of the engineers. [Docker](https://docs.docker.com/desktop/) is a tool used to integrate software dependencies and allow developers to quickly spin up software builds in portable lightweight containers which provide consistent environments, ensuring applications run the same way across various platforms. Here are the steps to check that the environment is running correctly:
 
-1. Clone the repo into your working directory.
+1. Fork the repo
+
+   ![fork](../data/fork.jpg)
+   
+2. Clone the forked repo into your working directory.
 ```sh
-git clone git@github.com:bloomberg/bbit-learning-labs.git
+git clone git@github.com:username/bbit-learning-labs.git
```

Comment:
git@github.com:[username]/bbit-learning-labs.git

Reply:
or maybe the step before this should be get your SSH key with a screenshot of where to get it circling the green code  button as well as the SSH key

Reply:
3. Get your SSH key from your fork